lib/url_launcher.dart

```diff
@@ -11,6 +14,40 @@ import 'pages/user.dart';
 import 'stores/accounts_store.dart';
 import 'stores/config_store.dart';
 import 'util/goto.dart';
+import 'util/text_color.dart';
+
+///Launches a Chrome Custom Tab on Android or Safari View Controller on iOS
+Future<void> _launchCustomTab(BuildContext context, String link) async {
+  try {
+    await launch(
+      link,
+      customTabsOption: CustomTabsOption(
+        toolbarColor: Theme.of(context).colorScheme.primary,
+        enableDefaultShare: true,
+        enableUrlBarHiding: true,
+        showPageTitle: true,
+        animation: CustomTabsSystemAnimation.slideIn(),
+        extraCustomTabs: const <String>[
+          // ref. https://play.google.com/store/apps/details?id=org.mozilla.firefox
+          'org.mozilla.firefox',
+          // ref. https://play.google.com/store/apps/details?id=com.microsoft.emmx
+          'com.microsoft.emmx',
+        ],
+      ),
+      safariVCOption: SafariViewControllerOption(
+        preferredBarTintColor: Theme.of(context).colorScheme.primary,
+        preferredControlTintColor:
+            textColorBasedOnBackground(Theme.of(context).colorScheme.primary),
+        barCollapsingEnabled: true,
+        entersReaderIfAvailable: false,
+        dismissButtonStyle: SafariViewControllerDismissButtonStyle.close,
+      ),
+    );
+  } catch (e) {
+    // An exception is thrown if browser app is not installed on Android device.
+    debugPrint(e.toString());
```

Comment:
I see this in the example code of the library. 
Could you please convert this to show the snack bar like we do for other errors.

Or maybe fallback to external browser?

Reply:
Definitely, implemented both suggestions in theory even if I don't have a great way of testing it without uninstalling/disabling all my browsers.